butlarr/services/__init__.py

```diff
@@ -66,7 +78,7 @@ def _delete(self, endpoint, params={}):
             f"{self.api_url}/{endpoint}", params={"apikey": self.api_key, **params}
         )
 
-    def request(self, endpoint: str, *, action=Action.GET, params={}, fallback=None):
+    def request(self, endpoint: str, *, action=Action.GET, params={}, fallback=None, raw=False):
         r = None
```

Comment:
We should probably add an assert or something here to prevent users from using raw and fallback at the same time, as it would cause potentially unexpected behaviour :)